docs/osf/setup.rst

```diff
@@ -326,7 +329,15 @@ Installing libxml2 and libxslt
 Install node packages with ``npm``
 ----------------------------------
 
-``npm`` is used to install required Node.JS packages.
+``npm`` (node package manager) is used to install required Node.JS packages. To install ``npm``, you have to install ``node.js``, which includes ``npm``. `npmjs.org <http://blog.npmjs.org/post/85484771375/how-to-install-npm>`_ recommends doing this through the `node.js installer <https://nodejs.org/en/>`_. Be sure to check for upgrades after installation by running: 
+
+    ::
+
+        sudo npm install npm -g
```

Comment:
Is this the recommended way to update npm? `sudo` is generally discouraged when using Homebrew.